src/enums.rs

```diff
@@ -5,2279 +5,2321 @@
 #[allow(non_upper_case_globals)]
 pub mod key {
     use gdk_sys;
+    use glib::GString;
+    use keys;
+    use glib::translate::*;
 
-    pub type Key = u32;
+    #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct Key(u32);
```

Comment:
```
    |                                                ^^^^^^ `gdk::enums::key::Key` cannot be formatted with the default formatter
```

This needs some fixing though. Can you impl the `Display` trait?

And imho all this should be in `src/key.rs`, let's not pollute the enum file with more things than necessary :)

Reply:
@danigm?

Reply:
I've moved the `Key` struct to the `keys` module and leave the `enums` module as it was. I've also implemented the `Display` trait, but I don't know what to write there, I've just done the same that the `Debug` does. Maybe we can print the unicode, but we should fallback to something for non unicode keys, or the value in hexadecimal, but I'm not sure what will be the use case for the `Display` trait.

Reply:
You still left the `pub type Key = u32;` and all the key constants as they are. Can you remove that and move the key constants over to the new file too while making them of the new `Key` type like you did before? Having two types for the same thing is rather suboptimal.

Apart from that this looks good to me now.